uint8 commandId
float32 setValue